namespaces/decentraland.md

```diff
@@ -0,0 +1,58 @@
+---
+namespace: urn:decentraland
+github-org: https://github.com/decentraland
+contact-email: developers@decentraland.org
+---
+
+# Description
+
+Decentraland is a virtual reality platform powered by the Ethereum blockchain. 
+
+# Resolver structure
+
+The structure of the resolver is defined by a hierarchy of content address resolvers.
+
+```yml
+urn:decentraland:{protocol}:(...)
+```
+
+The hierarchy is defined by the `{protocol}` component of the URN.
+
+## The Ethereum resolvers
+
+Decentraland works in Ethereum mainnet, ropsten and Matic. There are 4 registered roots for blockchain assets:
+
+- Ethereum Mainnet: `urn:decentraland:ethereum:{contract-or-alias}:{tokenId}`
+- Ethereum Ropsten: `urn:decentraland:ropsten:{contract-or-alias}:{tokenId}`
+- Matic Mainnet: `urn:decentraland:matic:{contract-or-alias}:{tokenId}`
+- Matic Mumbai: `urn:decentraland:matic:{contract-or-alias}:{tokenId}`
```

Comment:
Should it be  `urn:decentraland:mumbai:{contract-or-alias}:{tokenId}` ?

Reply:
typo